README.md

```diff
@@ -1,4 +1,61 @@
 # Dcrtimestamptweet
 
-> Twitter bot to timestamp a thread when @dcrtimestamptweet is mentioned.
+Twitter bot to timestamp a thread when a chosen keyword is mentioned. This bot uses IPFS as a decentralized database to store the tweet thread json and uses [dcrtime](https://github.com/decred/dcrtime) for timestamping.
 
+
+## Installation
+
+1. Create a twitter account and [apply for access](https://developer.twitter.com/en/apply-for-access.html)
+
+2. [Generate twitter access tokens](https://developer.twitter.com/en/docs/basics/authentication/guides/access-tokens.html)
+
+3. Make a .env File
+    ```
+    $ touch .env
+    ```
+
+4. Add the generated credentials in the `.env` file
+    ```
+    TWITTER_CONSUMER_KEY=<GENERATED_API_KEY>
+    TWITTER_CONSUMER_SECRET=<GENERATED_API_KEY_SECRET>
+    TWITTER_ACCESS_TOKEN=<GENERATED_ACCESS_TOKEN>
+    TWITTER_ACCESS_TOKEN_SECRET=<GENERATED_ACCESS_TOKEN_SECRET>
+    TRACKED_WORD="@dcrtimestamptweet" // mention which will call the bot
+    ```
+
+5. Run yarn command
+    ```
+    $ yarn
+    ```
+
+6. Start app
+    ```
+    yarn start
```

Comment:
You should add the `$` before the command here or remove it from other commands for consistency.